.github/workflows/build_test.yml

```diff
@@ -49,3 +49,24 @@ jobs:
             Services, ./packages/services/coverage/coverage-summary.json
             UNTP test suite, ./packages/untp-test-suite/coverage/coverage-summary.json
             VC test suite, ./packages/vc-test-suite/coverage/coverage-summary.json
+
+  build_docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: yarn
```

Comment:
Should we increase the Node version to 21 for consistency across this entire workflow file?

Reply:
What version is the app using? I.e. specified in the `.nvmrc` and `readme.md`?

Reply:
It’s at least 20.12.2. You’re right, node-version: 20 is fine.